crs/v1alpha2/controlplane_ubuntu.yaml

```diff
@@ -81,16 +81,27 @@ spec:
                   192.168.111.249
               }
           }
-      - path: /etc/network/interfaces
+          vrrp_instance VI_2 {
```

Comment:
Why is there a second vrrp config?

Reply:
This ensures the Ironic Endpoint IP 172.22.0.2 is also alive in target-cluster controlplane. This is added in 03.. scripts in minikube and this IP needs to be also on the controlplane so that after pivoting the ironic endpoint is reachable.

Reply:
This is problematic because as long as you have Minikube cluster still up and running, you will have ip address conflict